CONTRIBUTING.md

```diff
@@ -1,139 +1,123 @@
-to comment on an issue that you feel either is not a problem that needs to be
-fixed, or if you encounter information in an issue that you feel is incorrect,
-explain why you feel that way with additional supporting context, and be willing
-to be convinced that you may be wrong. By doing so, we can often reach the
-correct outcome much faster.
+Contributors are encouraged to help one another make forward progress as much as possible,
+empowering one another to solve issues collaboratively. If you choose to comment on an issue that
+you feel either is not a problem that needs to be fixed, or if you encounter information in an issue
+that you feel is incorrect, explain why you feel that way with additional supporting context, and be
+willing to be convinced that you may be wrong. By doing so, we can often reach the correct outcome
+much faster.
 
 ### Resolving a Bug Report
 
-In the majority of cases, issues are resolved by opening a Pull Request. The
-process for opening and reviewing a Pull Request is similar to that of opening
-and triaging issues, but carries with it a necessary review and approval
-workflow that ensures that the proposed changes meet the minimal quality and
-functional guidelines of the Tokio project.
+In the majority of cases, issues are resolved by opening a Pull Request. The process for opening and
+reviewing a Pull Request is similar to that of opening and triaging issues, but carries with it a
+necessary review and approval workflow that ensures that the proposed changes meet the minimal
+quality and functional guidelines of the Tokio project.
 
 ## Pull Requests
 
-Pull Requests are the way concrete changes are made to the code, documentation,
-and dependencies in the Tokio repository.
+Pull Requests are the way concrete changes are made to the code, documentation, and dependencies in
+the Tokio repository.
 
-Even tiny pull requests (e.g., one character pull request fixing a typo in API
-documentation) are greatly appreciated. Before making a large change, it is
-usually a good idea to first open an issue describing the change to solicit
-feedback and guidance. This will increase the likelihood of the PR getting
-merged.
+Even tiny pull requests (e.g., one character pull request fixing a typo in API documentation) are
+greatly appreciated. Before making a large change, it is usually a good idea to first open an issue
+describing the change to solicit feedback and guidance. This will increase the likelihood of the PR
+getting merged.
 
 ### Cargo Commands
 
-Due to the extensive use of features in Tokio, you will often need to add extra
-arguments to many common cargo commands. This section lists some commonly needed
-commands.
+Due to the extensive use of features in Tokio, you will often need to add extra arguments to many
+common cargo commands. This section lists some commonly needed commands.
 
 Some commands just need the `--all-features` argument:
 
-```
+```shell
```

Comment:
In general, I've gone with `shell` over `bash` as the commands are relevant generally to any shell.